dev-go/delve/delve-1.7.1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit go-module
+
+DESCRIPTION="A source-level debugger for the Go programming language"
+HOMEPAGE="https://github.com/go-delve/delve"
+
+EGO_PN="github.com/go-delve/delve"
+SRC_URI="https://${EGO_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~amd64"
+
+LICENSE="MIT BSD BSD-2 Apache-2.0"
+SLOT="0"
+IUSE="test"
+FEATURES=network-sandbox
```

Comment:
NNope, this won't do. Can't you somehow add `test? ( ... )` to `SRC_URI`?
You could also try adding `PROPERTIES="test_network"` here and maybe `go test` in `src_test` then works on its own?

Reply:
Unfortunately, `go test` needs network access to download more packages. I am still not familiar with go ebuilds. 
I will have to check how to specify these packages as test dependency and allow them to install.

Reply:
Just try it with the `PROPERTIES="test_network" ` I mentioned above, it was created for situations like this.

But yes, overall packaging go is a bit messy.

Reply:
I got a good help in the IRC channel and got everything working without networking. Just need to commit and push.